logstash/config/logstash.yml

```diff
@@ -9,3 +9,8 @@ http.host: "0.0.0.0"
 ##     https://www.elastic.co/guide/en/x-pack/current/installing-xpack.html#xpack-enabling
 #
 xpack.monitoring.enabled: false
+
+## The path to the Logstash config for the main pipeline.
+## see https://www.elastic.co/guide/en/logstash/current/logstash-settings-file.html
+#
+path.config: /usr/share/logstash/pipeline
```

Comment:
Oh sorry! It was added as part of elastic/logstash-docker@303eafecbd84753d42000772aafa14c1d1eec1ff, you're right. Can you just delete the comment then?